omero/util_scripts/Move_Annotations.py

```diff
@@ -76,28 +76,28 @@ def get_key(ann_link, with_owner=False):
 
     links_dict = {}
     # Remove duplicate annotations according to get_key(l)
-    for l in old_links:
-        links_dict[get_key(l, conn.isAdmin())] = l
+    for link in old_links:
+        links_dict[get_key(link, conn.isAdmin())] = link
 
     old_links = links_dict.values()
 
     # Find existing links on Well so we don't try to duplicate them
     existing_well_links = list(conn.getAnnotationLinks('Well', [well.id],
                                                        ns=ns, params=params))
-    existing_well_keys = [get_key(l) for l in existing_well_links]
+    existing_well_keys = [get_key(link) for link in existing_well_links]
 
     new_links = []
-    for l in old_links:
-        if get_key(l) in existing_well_keys:
+    for link in old_links:
+        if get_key(link) in existing_well_keys:
             continue
-        log("    Annotation: %s %s" % (l.child.id.val,
-                                       l.child.__class__.__name__))
+        log("    Annotation: %s %s" % (link.child.id.val,
+                                       link.child.__class__.__name__))
         link = WellAnnotationLinkI()
         link.parent = WellI(well.id, False)
-        link.child = l.child
```

Comment:
Isn't the test problem here related to the fact that you renamed the variable as `link` but the logic of loop also defines a `link` objet?

This is a no-op for instance

Reply:
well spotted. so many changes I missed it. I will check that

Reply:
test reactivated and everything is green